app/controllers/api/v1/registrations_controller.rb

```diff
@@ -0,0 +1,6 @@
+module Api
+  module V1
+    class RegistrationsController < DeviseTokenAuth::RegistrationsController
```

Comment:
Am not sure if you have to `before_action skip_session_storage` or `skip_before_action verify_authenticity_token`. Same applies to all devise controllers.